File: 2210-find-target-indices-after-sorting-array/find-target-indices-after-sorting-array.cpp
class Solution {
public:
    vector<int> targetIndices(vector<int>& nums, int target) {
        int l = 0, t = 0;
        vector<int> res;
        for(int i:nums){
            if(i<target) l++;
            else if(i == target)t++;
        }
        for(int i=l;i<l+t;i++)res.push_back(i);
        return res;
    }
};

File: 2210-find-target-indices-after-sorting-array/find-target-indices-after-sorting-array.py
class Solution:
    def targetIndices(self, nums: List[int], target: int) -> List[int]:
        lower = 0
        higher = 0
        length = len(nums)
        for i in nums:
            if i < target:
                lower += 1
            elif i > target:
                higher += 1
        return [i for i in range(lower, length - higher)]